go.mod

```
@@ -3,19 +3,24 @@ module github.com/giantswarm/capi-migration
 go 1.16
 
 require (
-	github.com/giantswarm/backoff v0.2.0
+	github.com/giantswarm/apiextensions/v3 v3.22.0
 	github.com/giantswarm/certs/v3 v3.0.0
 	github.com/giantswarm/k8sclient/v4 v4.0.0
 	github.com/giantswarm/microerror v0.3.0
 	github.com/giantswarm/micrologger v0.5.0
 	github.com/giantswarm/tenantcluster/v3 v3.0.0
-	github.com/onsi/ginkgo v1.14.1
-	github.com/onsi/gomega v1.10.2
+	github.com/onsi/ginkgo v1.14.2
+	github.com/onsi/gomega v1.10.3
 	k8s.io/api v0.18.9
 	k8s.io/apimachinery v0.18.9
 	k8s.io/client-go v0.18.9
-	sigs.k8s.io/cluster-api v0.3.7
+	sigs.k8s.io/cluster-api v0.3.14
+	sigs.k8s.io/cluster-api-provider-azure v0.4.13
```

Let's make this clear:

```suggestion
	sigs.k8s.io/cluster-api v0.0.0
	sigs.k8s.io/cluster-api-provider-azure v0.0.0
```

---

This works only partially:
```
$ git diff
diff --git a/go.mod b/go.mod
index 61a6b79..eae789c 100644
--- a/go.mod
+++ b/go.mod
@@ -14,8 +14,8 @@ require (
        k8s.io/api v0.18.9
        k8s.io/apimachinery v0.18.9
        k8s.io/client-go v0.18.9
-       sigs.k8s.io/cluster-api v0.3.14
-       sigs.k8s.io/cluster-api-provider-azure v0.4.13
+       sigs.k8s.io/cluster-api v0.0.0
+       sigs.k8s.io/cluster-api-provider-azure v0.0.0
        sigs.k8s.io/controller-runtime v0.6.3
        sigs.k8s.io/yaml v1.2.0
 )
```

```
$ go mod tidy
```
```
$ git diff
diff --git a/go.mod b/go.mod
index 61a6b79..51aa149 100644
--- a/go.mod
+++ b/go.mod
@@ -14,8 +14,8 @@ require (
        k8s.io/api v0.18.9
        k8s.io/apimachinery v0.18.9
        k8s.io/client-go v0.18.9
-       sigs.k8s.io/cluster-api v0.3.14
-       sigs.k8s.io/cluster-api-provider-azure v0.4.13
+       sigs.k8s.io/cluster-api v0.3.13
+       sigs.k8s.io/cluster-api-provider-azure v0.0.0
        sigs.k8s.io/controller-runtime v0.6.3
        sigs.k8s.io/yaml v1.2.0
 )
```

I'll add the result still.